docs/development.md

```diff
@@ -56,3 +56,12 @@ To debug on an Android device, please refer to the following steps:
 - Follow instructions in [Remote Debugging on Android](https://developer.chrome.com/devtools/docs/remote-debugging-legacy) to launch `adb`. Make sure to use port 9000 so that the existing debug configuration works.
 - in VSCode, select debug configuration `Remote Browser via Webkit Adaptor`.
 - follow the steps above to debug.
+
+### Performance profiling
+To profile the kernel level execution time, please run the following command:
+npm test -- model <model_name> --profile
+
+It generates raw perf data for each kernel, you may want use tools/parse-profiler.ts to parse it, e.g.:
+
+npm test -- model resnet50 -b=webgl --perf --profile > resnet50_perf.txt
+node tools/parse-profiler.js < resnet50_perf.txt > parsed_resnet50_perf.txt
```

Comment:
Looks like CI is failing complaining formatting this file.

Reply:
fixed.